README.md

```diff
@@ -151,7 +151,7 @@ To start Parity Ethereum as a regular user using `systemd` init:
 
 ## 4. Testing <a id="chapter-004"></a>
 
-You can run tests with the following commands:
+Download the required test files: `git submodule update --init --recursive`. You can run tests with the following commands:
```

Comment:
Does `cargo test --all` fail without this? I thought it was needed only for `cargo test --all --features=json-tests` but maybe I'm wrong?

Reply:
Yes, it fails we have [this macro](https://github.com/paritytech/parity-ethereum/blob/master/rpc/src/v1/tests/mod.rs#L31-#L41) which requires the `ethereum tests submodule` which is used in a couple of `rpc tests`

Reply:
I don't know if with `--all`, but it did for me when running `cargo test -p parity-json`